### ISSUE TITLE
Primary/Secondary companion instance option

### ISSUE THREAD

**Issue by Julusian** [Julusian/companion-remote#11]
It would be useful to be able to switch the companion ip in a couple of clicks.
This will allow for primary/backup flows, and other workflows

**Comment by Julusian** [comment]
I dont think that is a good idea.
Satellite has no idea what page it is currently on, so it will be relying on both companion machines keeping the page changes in sync. Meaning that if either you make changes in one and not the other, or the connection drops temporarily to one of them, then your streamdeck could end up on page 1 on A, and page 5 on B.

Also, if it is sending to both, does that mean that both companions will be expected to execute the actions for every press? That too will cause issues for anyone who has any buttons set to 'toggle' or similar behaviour

**Comment by MeestorX** [comment]
Fair enough. Trying to sort out how to set up a redundant system with the SD at a distance. Not easy!
